integration_test/third_party_apps_data/applications/nvml/exercise

```diff
@@ -0,0 +1,91 @@
+# build and run a test application that generates CUDA traffic
+
+tee test_cuda_kernel.cc > /dev/null << EOF
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// adapted from StackOverflow (license CC BY-SA 3.0)
```

Comment:
You need to include attribution for CC BY-SA ("If supplied, you must provide the name of the creator and attribution parties, a copyright notice, a license notice, a disclaimer notice, and a link to the material. CC licenses prior to Version 4.0 also require you to provide the title of the material if supplied, and may have other slight differences.")

Reply:
Replaced with a cuda demo suite app.